Issue #36 Fixes installation for Unix

kishansagathiya opened the pull request:
Current unix installation skip `src` repo and directly puts
`github.com/ipfs/ipfs-cluster` repo in $GOPATH. This commit fixes that
by go getting instead of cloning.

Fixes #36

hsanjuan commented on content/documentation/download.md:
````diff
@@ -52,8 +52,8 @@ The following requirements apply to the installation from source:
 ```
-git clone https://github.com/ipfs/ipfs-cluster.git $GOPATH/github.com/ipfs/ipfs-cluster
-cd $GOPATH/github.com/ipfs/ipfs-cluster
+go get -u -d github.com/ipfs/ipfs-cluster
````
Sorry, this is wrong. This will still download all deps, even if it won't install the application. We don't need to download all deps because we use `gx` for that.

kishansagathiya replied:
I will just add src then

kishansagathiya replied:
@hsanjuan Changed. Sorry for force push.

hsanjuan reviewed (changes requested):
Yes, the `src/` is missing.

kishansagathiya committed:
Issue #36 Fixes installation for Unix

Current unix installation skip `src` repo and directly puts
`github.com/ipfs/ipfs-cluster` repo in $GOPATH.
This commit fixes that.

License: MIT
Signed-off-by: Kishan Mohanbhai Sagathiya <kishansagathiya@gmail.com>

hsanjuan reviewed (approved):
THANKS!!
